pdks/Gf180/setup.py

```diff
@@ -0,0 +1,32 @@
+"""
```

Comment:
We should use a declarative pyproject.toml instead.  The setup.py method is now deprecated.

Reply:
Good point:
- https://github.com/dan-fritchman/Hdl21/issues/144